tests/models/gptsan_japanese/test_modeling_gptsan_japanese.py

```diff
@@ -0,0 +1,428 @@
+# coding=utf-8
+# Copyright 2023 GPTSANJapanese Authors and HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unittest
+
+import numpy as np
+
+from transformers import (
+    GPTSANJapaneseConfig,
+    GPTSANJapaneseForConditionalGeneration,
+    GPTSANJapaneseModel,
```

Comment:
Since only those two models are tested, they should be the only models in the doc/main init.